src/net.cpp

```diff
@@ -1865,6 +1877,7 @@ void CConnman::CreateNodeFromAcceptedSocket(std::unique_ptr<Sock>&& sock,
                              CNodeOptions{
                                  .permission_flags = permission_flags,
                                  .prefer_evict = discouraged,
+                                 .forced_inbound = forced,
```

Comment:
311902f2cf94ee0e11ed34a1373db42dbc218b20

should this be exposed in peer info?

Reply:
good idea, added a new commit for this